src/sites/link/clictune.com.js

```diff
@@ -1,11 +1,11 @@
 _.register({
   rule: {
     host: /^(www\.)?clictune\.com$/,
-    path: /^\/id=\d+/,
+    path: /^/[/^/]+$/,
```

Comment:
You should escape `/` because it's the terminal character of a regular expression, and there is an extra `/` after `[`.
We probably need `/^\/[^\/]+$/` here.